src/modm/platform/timer/stm32/advanced.hpp.in

```diff
@@ -338,6 +338,12 @@ public:
 		TIM{{ id }}->BDTR = flags;
 	}
 
+	static inline void
+	setRepetitionCount(uint32_t repetitionCount)
+	{
+		TIM{{ id }}->RCR = repetitionCount - 1;
```

Comment:
```suggestion
		TIM{{ id }}->RCR = repetitionCount;
```
Nah, let's not be too smart about this, adding a value of 0 underflows to 255 and that's weird. We should just reflect the RM functionality, you have to read it anyways to understand how to use the repetition counter.

Reply:
done

Reply:
(You can just resolve the conversation, it's like a shared todo list)